package.json

```diff
@@ -74,7 +75,7 @@
     "@testing-library/user-event": "^14.4.3",
     "@types/jest": "^29.5.1",
     "@types/mdx": "^2.0.5",
-    "@types/node": "^18.16.3",
+    "@types/node": "18.16.10",
```

Comment:
I've no idea why this changed. I only updated the lines for `cross-env`, cleaned `node_modules` and reran `npm install`.

Reply:
This change was caused by `npm run deploy`:
```console
$ npm run deploy

> nodejs.org@1.0.0 deploy
> npm run build && npm run export


> nodejs.org@1.0.0 build
> cross-env NODE_ENV=production NODE_NO_WARNINGS=1 next build

- info Linting and checking validity of types .It looks like you're trying to use TypeScript but do not have the required package(s) installed.
Installing dependencies

If you are not trying to use TypeScript, please remove the tsconfig.json file from your package root (and any TypeScript files in your pages directory).

- error ESLint must be installed in order to run during builds: npm install --save-dev eslint
- info Linting and checking validity of types ...
Installing devDependencies (npm):
- @types/node
```

Reply:
Super weird. This sounds odd. Do you want to make another hot fix PR or you want me to investigate it?

Reply:
I have no idea if we need to be constantly updating `@types/node` -- today it was bumped to 18.16.12 (yesterday it was bumped to 18.16.10). For this PR I'd reverted the change (so it did not change). I'll open an issue in case someone else wants to investigate.